closed/custom/Images-pre.gmk

```diff
@@ -18,4 +18,8 @@
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
 # ===========================================================================
 
-JRE_MODULES += openj9.cuda openj9.gpu
+JRE_MODULES += \
+	openj9.cuda \
+	openj9.dataaccess \
+	openj9.gpu \
+	#
```

Comment:
Why the final separator and `#` line?

Reply:
It's a pattern common in openjdk. I think the motivation is that to add to the list you don't have to change any existing lines.